### PR TITLE
add new constructor to QueryParams to set datacenter and index/waitTime

### PR DESCRIPTION
**Pull request by senthilkumar-kj**
Fixes https://github.com/Ecwid/consul-api/issues/79

### CODE REVIEW

**Comment by vgv** [comment]
Thank you!

File: src/main/java/com/ecwid/consul/v1/QueryParams.java [hunk 1]
```diff
@@ -47,6 +47,13 @@ public QueryParams(long waitTime, long index) {
 		this.index = index;
 	}
 
+    public QueryParams(String datacenter, long waitTime, long index) {
```
Comment (spencergibb) [inline src/main/java/com/ecwid/consul/v1/QueryParams.java]: Formatting, use tabs instead of spaces.

Comment (spencergibb) [inline src/main/java/com/ecwid/consul/v1/QueryParams.java]: Why not go with a builder pattern since every combination of fields would be unruly.

Comment (senthilkumar-kj) [inline src/main/java/com/ecwid/consul/v1/QueryParams.java]: Agree, we could do the same for PutParams. Next time, when I make a change, will change it to builder pattern.

Comment (senthilkumar-kj) [inline src/main/java/com/ecwid/consul/v1/QueryParams.java]: Noted, thanks.

**Pull request merged**